app/views/overview/_service-instance-row.html

```diff
@@ -6,8 +6,8 @@
     <div class="list-pf-content">
       <div class="list-pf-name">
         <h3>
-          <span ng-bind-html="row.displayName | highlightKeywords : row.state.filterKeywords"></span>
-          <div ng-bind-html="row.apiObject.metadata.name | highlightKeywords : row.state.filterKeywords" class="list-row-longname"></div>
```

Comment:
We should remove the `list-row-longname` styles from _overview.less file since this is the only place it's used.